### PR TITLE
Add Camera Shutter as optional parameter to config file

### PR DESCRIPTION
**Pull request by zsteed**
For those interested in having a sound effect when the camera action button is tapped, it defaults to false.

### CODE REVIEW

File: Source/CameraView/CameraView.swift [hunk 1]
```diff
@@ -218,7 +218,8 @@ class CameraView: UIViewController, CLLocationManagerDelegate, CameraManDelegate
 
   func takePicture(_ completion: @escaping () -> Void) {
     guard let previewLayer = previewLayer else { return }
-
+    SoundEffect.cameraShutter.play()
```
Comment (weslindsay) [inline Source/CameraView/CameraView.swift]: Should this be wrapped in a check against the configuration? 

```Swift
if configuration.addCameraSoundEffect {
  SoundEffect.cameraShutter.play()
}

```

File: Source/SoundEffect.swift [hunk 2]
```diff
@@ -0,0 +1,18 @@
+//
+//  SoundEffect.swift
+//  ImagePicker
+//
+//  Created by Zachary Steed on 9/1/17.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
+//
```
Comment (weslindsay) [inline Source/SoundEffect.swift]: Maybe remove this boilerplate?

File: Source/CameraView/CameraView.swift (hunk 1 above)
Comment (zsteed) [reply Source/CameraView/CameraView.swift]: Oh yeah, good catch. I should really review more code more often 👍

File: Source/SoundEffect.swift (hunk 2 above)
Comment (zenangst) [reply Source/SoundEffect.swift]: Yeah I would remove this

Comment (zenangst) [reply Source/SoundEffect.swift]: But I can't see this in the overall diff. Maybe GitHub is playing tricks on me.

**Pull request closed without merging**